drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/editor/strategies/KogitoDMNDataManagementStrategy.java

```diff
@@ -16,63 +16,64 @@
 import org.drools.workbench.screens.scenariosimulation.client.rightpanel.TestToolsView;
-import org.drools.workbench.screens.scenariosimulation.kogito.client.dmn.KogitoDMNService;
+import org.drools.workbench.screens.scenariosimulation.kogito.client.dmn.ScenarioSimulationKogitoDMNDataManager;
+import org.drools.workbench.screens.scenariosimulation.kogito.client.services.ScenarioSimulationKogitoDMNMarshallerService;
 import org.drools.workbench.screens.scenariosimulation.model.typedescriptor.FactModelTuple;
+
 import org.jboss.errai.common.client.api.ErrorCallback;
-import org.jboss.errai.common.client.api.RemoteCallback;
-import org.kie.workbench.common.dmn.webapp.kogito.marshaller.js.model.MainJs;
-import org.kie.workbench.common.dmn.webapp.kogito.marshaller.js.model.callbacks.DMN12UnmarshallCallback;
 import org.kie.workbench.common.dmn.webapp.kogito.marshaller.js.model.dmn12.JSITDefinitions;
 import org.uberfire.backend.vfs.Path;
 import org.uberfire.backend.vfs.PathFactory;
+import org.uberfire.client.callbacks.Callback;
 import org.uberfire.workbench.events.NotificationEvent;
 
 public class KogitoDMNDataManagementStrategy extends AbstractDMNDataManagementStrategy {
 
-    private KogitoDMNService dmnTypeService;
+    private ScenarioSimulationKogitoDMNDataManager dmnDataManager;
+    private ScenarioSimulationKogitoDMNMarshallerService dmnMarshallerService;
 
-    public KogitoDMNDataManagementStrategy(EventBus eventBus, KogitoDMNService dmnTypeService) {
+    public KogitoDMNDataManagementStrategy(EventBus eventBus,
+                                           ScenarioSimulationKogitoDMNDataManager dmnDataManager,
+                                           ScenarioSimulationKogitoDMNMarshallerService dmnMarshallerService) {
         super(eventBus);
-        this.dmnTypeService = dmnTypeService;
+        this.dmnDataManager = dmnDataManager;
+        this.dmnMarshallerService = dmnMarshallerService;
     }
 
     @Override
-    protected void retrieveFactModelTuple(TestToolsView.Presenter testToolsPresenter, ScenarioSimulationContext context, GridWidget gridWidget, String dmnFilePath) {
-        RemoteCallback<FactModelTuple> callback = getSuccessCallback(testToolsPresenter, context, gridWidget);
-        String fileName = dmnFilePath.substring(dmnFilePath.lastIndexOf('/') + 1);
-        final Path dmnPath = PathFactory.newPath(fileName, dmnFilePath);
-        dmnTypeService.getDMNContent(dmnPath, getDMNContentRemoteCallback(callback), getDMNContentErrorCallback(dmnFilePath));
-    }
+    protected void retrieveFactModelTuple(final TestToolsView.Presenter testToolsPresenter,
+                                          final ScenarioSimulationContext context,
+                                          final GridWidget gridWidget,
+                                          final String dmnFilePath) {
+        String dmnFileName = dmnFilePath.substring(dmnFilePath.lastIndexOf('/') + 1);
```

Comment:
@jstastny-cz It's safe for Windows - please check this PR https://github.com/kiegroup/kogito-tooling/pull/203 where I attached a screenshots which show how the path are shown in Linux and Windows environment.
As you can see, path are Unix-like in Windows as well, this is because ResourceContentAPI always returns path in unix-like pattern, event with a Windows environment.